test.js

```diff
@@ -28,3 +29,18 @@ napazone.execute((sharable) => {
 }, [napa.memory.crtAllocator]).then((r) => {
     console.log('...3...zone execute callback......', r.value, global.__zone_id, global.__worker_id);
 });
+
+napazone.on('Recycling', () => {
```

Comment:
use lower camel case

Reply:
fixed.

---
In reply to: [186880561](https://github.com/helloshuangzi/napajs/pull/20#discussion_r186880561) [](ancestors = 186880561)